BlueShift-iOS-SDK/BlueshiftRoutes.m

```diff
@@ -0,0 +1,59 @@
+//
+//  BlueshiftRoutes.m
+//  BlueShift-iOS-SDK
+//
+//  Created by Ketan Shikhare on 15/09/21.
+//
+
+#import <Foundation/Foundation.h>
+#import "BlueshiftRoutes.h"
+#import "BlueShift.h"
+#import "BlueShiftConfig.h"
+
+@implementation BlueshiftRoutes
+
++ (NSString*)getBaseURLString {
+    NSString *baseURL = kBS_USBaseURL;
+    switch (BlueShift.sharedInstance.config.region) {
+        case BlueshiftRegionEU:
+            baseURL = kBS_EUBaseURL;
+            break;
+            
+        default:
```

Comment:
do we need the default section here? we already have a default value assigned above.